components/_util/type.ts

```diff
@@ -1,3 +1,3 @@
 export type Diff<T extends string, U extends string> = ({ [P in T]: P } &
```

Comment:
Diff not used anywhere now.

Reply:
It's not used, but it's exported. Are we sure removing it won't break anyone's build?

Reply:
It's not a public API, we can remove it.

Reply:
Coming right up 🚀